### PR TITLE
[fix bug 1436740] Decommission /teach/smarton/ pages

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Removes legacy `/teach/smarton/` pages and redirects to `/internet-health/`

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1436740

## Testing
- [ ] Make sure I didn't miss any stray references to any of the smarton pages?

### CODE REVIEW

**Comment by alexgibson** [comment]
Successful test run: https://ci.us-west.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/186/pipeline/

File: bedrock/settings/base.py [hunk 1]
```diff
@@ -1346,9 +1344,6 @@ def lazy_email_backend():
 CSP_SCRIPT_SRC = CSP_DEFAULT_SRC + (
     # TODO fix things so that we don't need this
     "'unsafe-inline'",
-    # TODO snap.svg.js passes a string to Function() which is
-    # blocked without unsafe-eval. Find a way to remove that.
-    "'unsafe-eval'",
```
Comment (alexgibson) [inline bedrock/settings/base.py]: @pmac this PR removes `snap.svg.js` from the repo. I guess we are now safe to remove `unsafe-eval` here? :)

Comment (pmac) [reply bedrock/settings/base.py]: Ooooooo. Yeah! Hopefully so! Lets try that in another pr. Good catch 🔥

Comment (alexgibson) [reply bedrock/settings/base.py]: Ok, will leave this in for now

**Comment by alexgibson** [comment]
@craigcook bump

**Pull request merged**